assets/assetpack.def

```diff
@@ -114,6 +114,7 @@
 ! codemirror.js
 < https://raw.githubusercontent.com/codemirror/CodeMirror/8097c7e75ce7ef0512debe9967d7568626106e53/lib/codemirror.js
 < https://raw.githubusercontent.com/codemirror/CodeMirror/8097c7e75ce7ef0512debe9967d7568626106e53/mode/perl/perl.js
+< https://raw.githubusercontent.com/codemirror/CodeMirror/8097c7e75ce7ef0512debe9967d7568626106e53/mode/yaml/yaml.js
```

Comment:
Not sure whether you're aware, but the cache for the rpm package needs to be updated as well (using `update-cache.sh` script in the package repo).

Reply:
You mean [update-cache.sh](https://build.opensuse.org/package/view_file/devel:openQA/openQA/update-cache.sh?expand=1) as referenced in [openQA.spec](../blob/master/openQA.spec)? When is this script called?

Reply:
I'm not 100 % sure of the workflow myself anymore. But I suppose you have to get a current checkout of the OBS repo locally and than simply call the update-cache.sh script manually. Then you can check in the new cache.txz it generated (after verifying that is is not broken).